cw/util/device.py

```diff
@@ -76,8 +76,9 @@ def initialize_fpga(self, fpga, bitstream, pll_frequency):
         # bitstream.
         # Unfortunately, this doesn't seem to work for the CW310 yet,
         # see https://github.com/lowRISC/ot-sca/issues/48.
+        # This should work now. Issue is closed
         # TODO: We should have this in the CLI.
-        force_programming = False
+        force_programming = True
```

Comment:
IIRC this will cause the FPGA to always be re-programmed which we actually don't want no?